platform/android/app/src/main/kotlin/coop/polypoly/polypod/PolyStyle.kt

```diff
@@ -0,0 +1,91 @@
+package coop.polypoly.polypod
+
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+data class PolyStyle(
+    val spacing: PolySpacing = PolySpacing(),
+    val radius: PolyRadius = PolyRadius(),
+    val font: PolyFont = PolyFont(),
+    val border: PolyBorder = PolyBorder()
+)
+
+data class PolySpacing(
+    val _1x: Dp = 4.dp,
+    val _2x: Dp = (2 * _1x.value).dp,
+    val _3x: Dp = (3 * _1x.value).dp,
+    val _4x: Dp = (4 * _1x.value).dp,
+    val _5x: Dp = (5 * _1x.value).dp,
+    val _6x: Dp = (6 * _1x.value).dp,
+    val _7x: Dp = (7 * _1x.value).dp,
+    val _8x: Dp = (8 * _1x.value).dp,
+)
+
+data class PolyRadius(
+    val _1x: Dp = 4.dp,
+    val _2x: Dp = (2 * _1x.value).dp,
+    val _3x: Dp = (3 * _1x.value).dp,
+    val _4x: Dp = (4 * _1x.value).dp,
+    val _5x: Dp = (5 * _1x.value).dp,
+    val _6x: Dp = (6 * _1x.value).dp,
+)
+
+data class PolyFontFamily(
+    val jostRegular: Int = R.font.jost_regular,
+    val jostMedium: Int = R.font.jost_medium,
+)
+
+data class PolyFontSize(
+    val xs: TextUnit = 12.sp,
```

Comment:
It's also using different units here, while it uses no units in swift. It probably makes sense, but no idea.

Reply:
Ah, OK, I see they're different types; Kotlin does not type the thing.

Reply:
All values in swift are already in points(aka DP for android), therefore no need to specify explicitly the units.